src/widgets/sort-by-selector/__tests__/__snapshots__/sort-by-selector-test.js.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`sortBySelector() calls twice ReactDOM.render(<Selector props />, container) 1`] = `
+<HeaderFooter-AutoHide
```

Comment:
displayName is wrong on all of them, but probably not a big deal

Reply:
What name were you expecting?

Reply:
`Selector` (which doesn't have a displayName, so we'd need to add it first)